ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__periodic.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  osde2e:
+    name: osde2e
+    namespace: ci
+    tag: latest
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-test-harness
+  commands: |
+    export REPORT_DIR="$ARTIFACT_DIR"
+    export CONFIGS="rosa,stage,test-harness"
+    export ROSA_ENV="stage"
+    export ROSA_STS="true"
+    export ROSA_AWS_REGION="random"
+    export TEST_HARNESSES="quay.io/rhobs/observability-operator-test-harness"
+    export CHANNEL="stable"
+    export SECRET_LOCATIONS="/usr/local/osde2e-common,/usr/local/osde2e-credentials,/usr/local/osde2e-rosa-stage"
+    /osde2e test --secret-locations ${SECRET_LOCATIONS} --configs ${CONFIGS}
```

Comment:
According to my test, locally run test-harness with the following scripts works, should change the commands accordingly
```
#!/usr/bin/env bash
set -x
export AWS_SECRET_ACCESS_KEY="$"
export AWS_ACCESS_KEY_ID="$"
export OCM_TOKEN="$" 
CLUSTER_ID="23u83n9fp9sldmm0eq00f084hkobu2g2"   \
TEST_HARNESSES="quay.io/sthaha/observability-operator-test-harness:latest" \
ROSA_ENV="stage" \
ROSA_AWS_REGION="us-west-2" \
CLOUD_PROVIDER_REGION="us-west-2" \
POLLING_TIMEOUT="1800" \
./out/osde2e test \
--configs rosa,stage,test-harness \
--must-gather true \
--skip-health-check true \
--skip-destroy-cluster true 
```